cli/args/flags.rs

```diff
@@ -5858,6 +5880,23 @@ mod tests {
     );
   }
 
+  #[test]
+  fn run_env_file() {
+    let r =
+      flags_from_vec(svec!["deno", "run", "--env-file=.env", "script.ts"]);
+    assert_eq!(
```

Comment:
Please add a test that checks that `deno run --env` will result in `.env` file being the default value.